hieradata/common.yaml

```diff
@@ -343,6 +343,7 @@ govuk::apps::cache_clearing_service::enabled: false
 govuk::apps::cache_clearing_service::nagios_memory_warning: 2750
 govuk::apps::cache_clearing_service::nagios_memory_critical: 3000
 govuk::apps::cache_clearing_service::puppetdb_node_url: 'http://puppetdb.cluster/v2/nodes'
+govuk::apps::cache_clearing_service::rabbitmq::enabled: false
```

Comment:
If the default is disabled, this is redundant right?

Reply:
Yes, but there is a script enforcing parity of parameters between hieradata and hieradata_aws, so it was necessary to add. Thanks for the review! : )